OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -170,6 +170,13 @@ public interface INotifyDockHost { void Docked(Actor self, Actor client); void U
 	[RequireExplicitImplementation]
 	public interface INotifyDockClient { void Docked(Actor self, Actor host); void Undocked(Actor self, Actor host); }
 
+	[RequireExplicitImplementation]
+	public interface INotifyDockClientMoving
```

Comment:
Why is this not part of `INotifyDockClient`?

Reply:
Because Harvester (nor other clients) isn't supposed to inherit it. It'a a notify inferface